File: api-tester/src/bc5SupportDev/java/com/revenuecat/apitester/java/PurchasesBC5API.java
package com.revenuecat.apitester.java;

import com.revenuecat.purchases.BillingFeature;

@SuppressWarnings({"unused"})
final class PurchasesBC5API {
    static void check(final BillingFeature feature) {
        switch (feature) {
            case SUBSCRIPTIONS:
            case SUBSCRIPTIONS_UPDATE:
            case PRICE_CHANGE_CONFIRMATION:
        }
    }
}

File: api-tester/src/bc5SupportDev/java/com/revenuecat/apitester/kotlin/PurchasesBC5API.kt
package com.revenuecat.apitester.kotlin

import com.revenuecat.purchases.BillingFeature

@Suppress("unused")
private class PurchasesBC5API {
    fun check(billingFeature: BillingFeature) {
        when (billingFeature) {
            BillingFeature.PRICE_CHANGE_CONFIRMATION,
            BillingFeature.SUBSCRIPTIONS,
            BillingFeature.SUBSCRIPTIONS_UPDATE -> {
            }
        }.exhaustive
    }
}
